### PR TITLE
Removes #1 cause of suicide in Lizards

### PR DESCRIPTION
**Pull request by Moltijoe**
image
image

Why do I do this to myself

:cl:  
rscadd: Lizards now regrow their tail after 30 minutes
/:cl:

### CODE REVIEW

**Comment by Moggram** [comment]
HA!

**Comment by Maia-J** [comment]
20. Take it or leave it

**Comment by Moltijoe** [comment]
> 20. Take it or leave it

leave it

**Comment by MajManatee** [comment]
30 does feel like a long time when i could just kill myself

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm [hunk 1]
```diff
@@ -89,6 +90,12 @@
 					H.emote("wag")
 				if(-1)
 					stop_wagging_tail(H)
+	if(!regrowtimer && !("waggingtail_lizard" in mutant_bodyparts) && !("tail_lizard" in mutant_bodyparts))
+		regrowtimer = addtimer(CALLBACK(src, .proc/regrow_tail), 30 MINUTES, TIMER_UNIQUE)
+
+/datum/species/lizard/proc/regrow_tail()
+	if(!("waggingtail_lizard" in mutant_bodyparts) && !("tail_lizard" in mutant_bodyparts))
+		mutant_bodyparts |= "tail_lizard"
```
Comment (MajManatee) [inline code/modules/mob/living/carbon/human/species_types/lizardpeople.dm:98]: maybe add a to_chat for this? Feels like there isnt enoug feedback.

Comment (tattax) [reply code/modules/mob/living/carbon/human/species_types/lizardpeople.dm:98]: a visible message would be better imo

**Comment by Marmio64** [comment]
just get it reattached?

**Review by MajManatee — APPROVED** [review]
lizard

**Comment by milk-is-cool** [comment]
lizard tail farm confirmed?

**Pull request merged**